itou/siae_evaluations/fixtures.py

```diff
@@ -0,0 +1,141 @@
+from itou.eligibility.models import AdministrativeCriteria
+from itou.eligibility.models.iae import EligibilityDiagnosis
+from itou.institutions.models import Institution
+from itou.job_applications.models import JobApplication, JobApplicationWorkflow
+from itou.siae_evaluations.models import EvaluationCampaign
+from itou.siaes.models import Siae
+from itou.users.models import User
+
+
+NAMES = [
+    ("Gustave", "Loiseau", "187104712403340", "703059J"),
+    ("Barnabé", "Loiseau", "187105714612783", "476261C"),
+    ("Oliveria", "Da Figueira", "187102017128676", "354471C"),
+    ("Fan", "Se-Yeng", "187102531808302", "145793B"),
+    ("Diego", "le Navarrais", "187101703338725", "718757J"),
+    ("Bianca", "Castafiore", "287103404102922", "50704F"),
+    ("Tryphon", "Tournesol", "187105012012584", "321089E"),
+    ("Bunji", "Kuraki", "187101300806442", "298096C"),
+    ("Roberto", "Rastapopoulos", "187107922316551", "424548G"),
+    ("Séraphin", "Lampion", "187104308500783", "491405D"),
+    ("Allan", "Thompson", "187101033733471", "391055C"),
+    ("Eugène", "Triboulet", "187107317012993", "103819B"),
+    ("Igor", "Wagner", "187107823832946", "621359J"),
+    ("Frank", "Wolff", "187106129833161", "644055J"),
+    ("Tristan", "Bior", "187106539602337", "355357E"),
+    ("Herbert", "Dawes", "187103415217885", "401486E"),
+    ("Émile", "Vanneau", "187102117208427", "354895F"),
+    ("Jean-Loup", "de La Batellerie", "187109413107670", "354895D"),
+]
+
+
+def generate_approval_number():
+    return f"{Approval.ASP_ITOU_PREFIX}{random.randint(0000000, 9999999)}"
+
+
+with override_settings(EMAIL_BACKEND="django.core.mail.backends.locmem.EmailBackend"):
+    today = timezone.localdate()
+    evaluated_period_start_at = today - relativedelta(months=3)
+    evaluated_period_end_at = today - relativedelta(months=1)
+    created_job_applications_pks = []
+    controlled_siae = Siae.objects.get(pk=2653)  # has an active convention
+    siae_first_member = controlled_siae.members.first()
+    total_administrative_criteria = AdministrativeCriteria.objects.count()
+    level_to_criteria_pks = dict(
+        AdministrativeCriteria.objects.values("level").annotate(pks=ArrayAgg("pk")).values_list("level", "pks")
+    )
+
+    users = User.objects.filter(
+        username__in=[f"siae_evaluations_{i}" for i in range(1, total_administrative_criteria)]
+    )
+    users.delete()
+    EvaluationCampaign.objects.all().delete()
+
+    # We can't use factories here because FactoryBoy is not installed in Review app and Demo environments.
+    for i in range(1, total_administrative_criteria):
+        job_seeker = User.objects.create(
+            first_name=NAMES[i - 1][0],
+            last_name=NAMES[i - 1][1],
+            username=f"siae_evaluations_{i}",
+            kind=users_enums.KIND_JOB_SEEKER,
+            nir=NAMES[i - 1][2],
+            pole_emploi_id=NAMES[i - 1][3],
+            title=random.choice(users_enums.Title.values),
+        )
```

Comment:
Comme tu veux, c’est plus lisible.
```suggestion
        first_name, last_name, nir, pe_id = NAMES[i - 1]
        job_seeker = User.objects.create(
            first_name=first_name,
            last_name=last_name,
            username=f"siae_evaluations_{i}",
            kind=users_enums.KIND_JOB_SEEKER,
            nir=nir,
            pole_emploi_id=pe_id,
            title=random.choice(users_enums.Title.values),
        )
```

Reply:
En effet ! :)